### DISCUSSION TITLE
Monitor scale

### DISCUSSION THREAD

**Discussion by nullptroma** [end-4/dots-hyprland#407] [Q&A]
How to change monitor scale correctly? I want to set 1.666667, but when I set it, windows renders on small part of the screen. AGS overview also shows that fullscreen windows not fill all space. Problem with the command `windowrule=move 100%-16 100%-16,title:^(Picture-in-Picture)$`, which places windows to right-bottom part which I can not see (screensshot)

![image](https://github.com/end-4/dots-hyprland/assets/64524860/efc07d96-105c-48ea-94ee-449e6bb54ffe)

**Comment by end-4** [comment]
The overview is like that because it doesn't account for hyprland's monitor scale
I don't know about the PIP window
